Eshop.Domain/Category.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
```

Comment:
These using are not used - even VS is helping you - remove them.
If you follow Product.cs everything will be easier.

Reply:
fixed, no more that stuff - same as Product.cs